solv/simplequery.c

```diff
@@ -132,3 +132,25 @@ SolvFindSolvablesByNevraStr(
 error:
     goto cleanup;
 }
+
+uint32_t
+SolvRequiresFromQueue(
```

Comment:
code duplication:

https://github.com/vmware/tdnf/blob/dev/solv/tdnfpackage.c#L2064

please use a single function

Reply:
I think it was my intention to do that, but it needs more changes. They are not quite the same - this one creates a `Queue` of solvable ids, `SolvGetDependenciesFromId` creates a list of names. `SolvGetDependenciesFromId` should call this one, or the code that uses it needs to be redesigned.